doc/releases/release-notes-2.4.rst

```diff
@@ -427,6 +451,8 @@ Drivers and Sensors
     selects that are not specified by a cs-gpios property.
   * Added driver for the Xilinx AXI Quad SPI IP
   * STM32: Various fixes around DMA mode.
+  * Extended MCUX Flexcomm driver to support slave mode.
```

Comment:
Missing SPI. Extended MCUX Flexcomm "SPI" drive

Reply:
I didn't include it because this is a sub-bullet in the SPI drivers section